app/user/user_main.c

```diff
@@ -187,60 +184,73 @@ void user_rf_pre_init(void)
 }
 #endif
 
-/******************************************************************************
- * FunctionName : user_rf_cal_sector_set
- * Description  : SDK just reversed 4 sectors, used for rf init data and paramters.
- *                We add this function to force users to set rf cal sector, since
- *                we don't know which sector is free in user's application.
- *                sector map for last several sectors : ABCCC
- *                A : rf cal
- *                B : rf init data
- *                C : sdk parameters
- * Parameters   : none
- * Returns      : rf cal sector
-*******************************************************************************/
-uint32
-user_rf_cal_sector_set(void)
+#define EAGLE_FLASH_BIN_ADDR      (SYSTEM_PARTITION_CUSTOMER_BEGIN + 1)
+#define EAGLE_IROM0TEXT_BIN_ADDR  (SYSTEM_PARTITION_CUSTOMER_BEGIN + 2)
+static const partition_item_t pt_template[] ICACHE_RODATA_ATTR STORE_ATTR = {
+    { EAGLE_FLASH_BIN_ADDR,     0x00000, 0x10000},
+    { EAGLE_IROM0TEXT_BIN_ADDR, 0x10000, 0x60000},
```

Comment:
Will your current firmware fit into `0x60000`?
The partition length should be calculated from `platform_flash_get_first_free_block_address()` to be on the safe side.

Reply:
Currently LFS and the TLS certificate allocate flash page-aligned areas.  Perhaps these need to be moved into the partition table?